### PR TITLE
Blog post on sea-level rise estimation respecting volume below sea-level and densities etc.

### PR DESCRIPTION
**Pull request by jmalles**
My commits are a bit confusing, sorry...

### CODE REVIEW

**Comment by jmalles** [comment]
My commits are a bit confusing, sorry...

File: _posts/2021-02-25-slr-bsl.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+aspects as, e.g., the different densities of ice, freshwater, and ocean water. Another issue is the fact that some of the global glacier ice is situated 
+below sea-level, which needs to be accounted for in order not to overestimate sea-level changes due to glacier mass loss. Moreover, ice of glaciers 
+that drain into the ocean is likely to produce icebergs. Because melt-/freshwater has a lower density than the ocean water that is displaced by the iceberg, 
+a small part of the sea-level rise that is produced by such icebergs will occur with a delay over the time they are melting. This is known as a 
+halosteric effect. A competing effect is the thermosteric one, which describes the fact that ice melt that is induced by extracting the necessary latent heat from 
+the ocean will cause a contraction of the ocean water due to the implied cooling [(Jenkins, A., and Holland, D., 2007)](#refs).
+
+In this [Jupyter Notebook](https://nbviewer.jupyter.org/gist/jmalles/ca70090812e6499b34a22a3a7a7a8f2a) I explain approaches to calculating sea-level rise from glacier mass loss,
```
Comment (fmaussion) [inline _posts/2021-02-25-slr-bsl.md:18]:
```suggestion
In this [Jupyter Notebook](https://nbviewer.jupyter.org/gist/jmalles/ca70090812e6499b34a22a3a7a7a8f2a) I compare various approaches to calculating sea-level rise from glacier mass loss, 
```

File: _posts/2021-02-25-slr-bsl.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+that drain into the ocean is likely to produce icebergs. Because melt-/freshwater has a lower density than the ocean water that is displaced by the iceberg, 
+a small part of the sea-level rise that is produced by such icebergs will occur with a delay over the time they are melting. This is known as a 
+halosteric effect. A competing effect is the thermosteric one, which describes the fact that ice melt that is induced by extracting the necessary latent heat from 
+the ocean will cause a contraction of the ocean water due to the implied cooling [(Jenkins, A., and Holland, D., 2007)](#refs).
+
+In this [Jupyter Notebook](https://nbviewer.jupyter.org/gist/jmalles/ca70090812e6499b34a22a3a7a7a8f2a) I explain approaches to calculating sea-level rise from glacier mass loss, 
+respecting the density and volume below sea-level issues described above. I put the links to the data and notebook below in case you want to play around with it yourself.
+
+Although the equations proposed in the Notebook should theoretically be sounder than a previously published one [(Farinotti et al., 2019)](#refs), the quantitative differences are
```
Comment (fmaussion) [inline _posts/2021-02-25-slr-bsl.md:21]:
```suggestion
The equations proposed in the Notebook should theoretically be somewhat sounder than a previously published one [(Farinotti et al., 2019)](#refs). The quantitative differences are 
```

File: _posts/2021-02-25-slr-bsl.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+that drain into the ocean is likely to produce icebergs. Because melt-/freshwater has a lower density than the ocean water that is displaced by the iceberg, 
+a small part of the sea-level rise that is produced by such icebergs will occur with a delay over the time they are melting. This is known as a 
+halosteric effect. A competing effect is the thermosteric one, which describes the fact that ice melt that is induced by extracting the necessary latent heat from 
+the ocean will cause a contraction of the ocean water due to the implied cooling [(Jenkins, A., and Holland, D., 2007)](#refs).
+
+In this [Jupyter Notebook](https://nbviewer.jupyter.org/gist/jmalles/ca70090812e6499b34a22a3a7a7a8f2a) I explain approaches to calculating sea-level rise from glacier mass loss, 
+respecting the density and volume below sea-level issues described above. I put the links to the data and notebook below in case you want to play around with it yourself.
+
+Although the equations proposed in the Notebook should theoretically be sounder than a previously published one [(Farinotti et al., 2019)](#refs), the quantitative differences are 
+quite small globally.
```
Comment (fmaussion) [inline _posts/2021-02-25-slr-bsl.md:22]:
```suggestion
quite small globally (2.4 mm SLR, 0.8%), but can reach 4% regionally.
```

**Review by fmaussion — COMMENTED** [review]
Thanks! I've made some suggestions

**Comment by fmaussion** [comment]
Here is also a sligthly tuned notebook: 

[slr_bsl.ipynb.zip](https://github.com/OGGM/oggm.github.io/files/6090788/slr_bsl.ipynb.zip)

You can update your gist with it. Thanks!

**Comment by jmalles** [comment]
Do the line breaks in the .md-file for the blog post matter?

File: _posts/2021-02-25-slr-bsl.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+that drain into the ocean is likely to produce icebergs. Because melt-/freshwater has a lower density than the ocean water that is displaced by the iceberg, 
+a small part of the sea-level rise that is produced by such icebergs will occur with a delay over the time they are melting. This is known as a 
+halosteric effect. A competing effect is the thermosteric one, which describes the fact that ice melt that is induced by extracting the necessary latent heat from 
+the ocean will cause a contraction of the ocean water due to the implied cooling [(Jenkins, A., and Holland, D., 2007)](#refs).
+
+In this [Jupyter Notebook](https://nbviewer.jupyter.org/gist/jmalles/ca70090812e6499b34a22a3a7a7a8f2a) I compare various approaches to calculating sea-level rise from glacier mass loss, 
+respecting the density and volume below sea-level issues described above. I put the links to the data and notebook below in case you want to play around with it yourself.
+
+The equations proposed in the Notebook should theoretically be somehwat sounder than a previously published one [(Farinotti et al., 2019)](#refs). Yet, the quantitative differences are
```
Comment (fmaussion) [inline _posts/2021-02-25-slr-bsl.md:21]:
```suggestion
The equations proposed in the Notebook should theoretically be a bit sounder than previously used in [Farinotti et al., 2019](#refs). Yet, the quantitative differences are 
```

**Comment by fmaussion** [comment]
Thanks!

**Pull request merged**